cyclonedx_py/parser/environment.py

```diff
@@ -43,19 +44,26 @@
 else:
     from importlib_metadata import metadata, PackageMetadata as _MetadataReturn
 
-from cyclonedx.model import LicenseChoice
+from cyclonedx.model import License, LicenseChoice
 from cyclonedx.model.component import Component
 from cyclonedx.parser import BaseParser
 
 
+@enum.unique
+class LicenseOutputFormat(enum.Enum):
+    EXPRESSION = 'expression'
+    LICENSE = 'license'
```

Comment:
:x: please UpperCalecase names
:x: named license should be made clearer
:x: even though this enum will eventually affect the output(CLI result), it affects the builder/factory results - therefore rename

```py
@enum.unique
class LicenseFormat(enum.Enum):
    Expression = 'license-expression'
    Name = 'named-license'
```